### PR TITLE
cancel muting should not exit select-mode

### PR DESCRIPTION
**Pull request by r10s**
exiting 'select mode' is unexpected when hitting 'cancel' - the user may still want to do other things with the selection.

this makes the 'mute' behavior consisten with other cases showing a dialog, eg. 'delete'

### CODE REVIEW

**Review by adbenitez — APPROVED** [review]
good improvement, didn't test

**Comment by r10s** [comment]
yeah, came over that while searching for a "Mark All As Read" :)

let's wait for merging until #3122 is merged ... rebasing this PR seems easier :)

**Comment by adbenitez** [comment]
it seems no rebasing is needed since the file was just moved but there is no content conflict :thinking:

**Comment by r10s** [comment]
github said there's a merge conflict, rebase then did not need interaction .... some say so and some say so :)

**Pull request merged**
